modules/storage-page-memory/src/main/java/org/apache/ignite/internal/storage/pagememory/mv/FindRowVersion.java

```diff
@@ -113,26 +113,27 @@ public void finish() {
     /**
      * Row version filter in the version chain.
      */
+    @FunctionalInterface
     interface RowVersionFilter {
         /**
          * Returns {@code true} if the version matches.
          *
          * @param rowVersionLink Row version link;
-         * @param rowVersionAdder Address to row version (including page address + offset within it).
+         * @param rowVersionAddr Address of row version (including page address + offset within it).
          */
-        boolean apply(long rowVersionLink, long rowVersionAdder);
+        boolean apply(long rowVersionLink, long rowVersionAddr);
 
         static RowVersionFilter equalsByTimestamp(HybridTimestamp timestamp) {
-            return (rowVersionLink, rowVersionAdder) -> {
-                HybridTimestamp readTimestamp = HybridTimestamps.readTimestamp(rowVersionAdder, RowVersion.TIMESTAMP_OFFSET);
+            return (rowVersionLink, rowVersionAddr) -> {
+                HybridTimestamp readTimestamp = HybridTimestamps.readTimestamp(rowVersionAddr, RowVersion.TIMESTAMP_OFFSET);
 
                 return readTimestamp != null && readTimestamp.compareTo(timestamp) == 0;
```

Comment:
Can this be a `Objects.equals(readTimestamp, timestamp)`?

Reply:
We can do this, I'll change it.